src/libraries/System.Private.Xml/src/System/Xml/Core/XmlTextReaderImpl.cs

```diff
@@ -2989,7 +3080,7 @@ private void InitFragmentReader(XmlNodeType fragmentType, XmlParserContext parse
                 if (parserContext.NamespaceManager != null)
                 {
                     _namespaceManager = parserContext.NamespaceManager;
-                    _xmlContext.defaultNamespace = _namespaceManager.LookupNamespace(string.Empty);
+                    _xmlContext.defaultNamespace = _namespaceManager.LookupNamespace(string.Empty)!;
```

Comment:
no, all the code assumes it's never null and also it's always added to the lookup table so for this particular entry it will never be null and therefore `!`